### ISSUE TITLE
Unable to process films/TV shows with commas in "title"

### ISSUE THREAD

**Issue by ikarosghoste** [Kometa-Team/Kometa#1586]
### Version Number

1.19.0

### What branch are you on?

master

### Describe the Bug

When attempting to add a film with a comma in the title, it never actually locates the film. There might already be a fix for this, but I also noticed that search behavior was modified in #100, so I wanted to check if this was going to be in conflict. An example of a film: "Crazy, Stupid, Love". This would need to be processed as one title, not 3, but If I am understanding correctly, #100 modified search behavior by parsing the search into 3 separate titles. 

Is there a simple way around this, other than defining an alt_title?

There is no thrown exception, it simply ignores the title and continues to process.

Regards,
K

**Comment by YozoraXCII** [comment]
Do you have an example of what it is you are trying to run?

If a string has a comma, you should wrap the string in quotation marks otherwise it is read as a list

**Comment by ikarosghoste** [comment]
I discovered that this was not actually the issue!~ I really appreciate the quick response!

I had a colleague check my work, and my deduction was incorrect. That same colleague happens to be a movie buff and pointed out my error in the title:

Correct title: "Crazy, Stupid, Love." A PERIOD!? I didn't think to actually check against the IMDB..
Another example that he was able to correct: "Dude, Where's my Car?" title had the comma and the apostrophe, but obviously not the question mark.

For posterity, I was using the following:
<collection_name>
    plex_search:
      any:
        title.is:
          - "Crazy, Stupid, Love" # was missing the period

P.S. I joined the discord, so if I run into similar issues - Ill ask there first before submitting an issue =)
